File: quantum-vulnerable/quantumVulnerable.js
// Quantum-vulnerable (Shor-breakable) public-key algorithms
const ALG_ECGDSA_0 = 'ECGDSA';
const ALG_FFDHE2048_1 = 'ffdhe2048';
const ALG_BRAINPOOLP256R1_2 = 'brainpoolP256r1';
const ALG_ECQV_3 = 'ECQV';
const ALG_RSAESOAEP_4 = 'RSAES-OAEP';
const ALG_RSA3072_5 = 'RSA-3072';
const ALG_ECSCHNORR_6 = 'EC-Schnorr';
const ALG_SECT283K1_7 = 'sect283k1';
const ALG_RSASSAPSS_8 = 'RSASSA-PSS';
const ALG_ECGOST_9 = 'EC-GOST';
const ALG_FFDHE8192_10 = 'ffdhe8192';
const ALG_ED25519PH_11 = 'Ed25519ph';
const ALG_TEXTBOOKRSA_12 = 'textbook-RSA';
const ALG_SECP256R1_13 = 'secp256r1';
// TODO migrate ffdhe2048 to a post-quantum scheme (harvest-now-decrypt-later)
// TODO migrate NIST-P256 to a post-quantum scheme (harvest-now-decrypt-later)
// TODO migrate P-256 to a post-quantum scheme (harvest-now-decrypt-later)
// TODO migrate RSAES-PKCS1-v1_5 to a post-quantum scheme (harvest-now-decrypt-later)
// TODO migrate Rabin-Williams to a post-quantum scheme (harvest-now-decrypt-later)
// TODO migrate RSA-768 to a post-quantum scheme (harvest-now-decrypt-later)
const enabledAlgorithms = ['ECMQV', 'RSA-1024', 'X25519', 'Cramer-Shoup', 'ffdhe8192', 'MODP-6144', 'textbook-RSA', 'Okamoto-Uchiyama'];

File: quantum-vulnerable/quantum_vulnerable.py
# Quantum-vulnerable (Shor-breakable) public-key algorithms
ALG_RSA1024_0 = "RSA-1024"
ALG_PRIME256V1_1 = "prime256v1"
ALG_DSA3072_2 = "DSA-3072"
ALG_DSA1024_3 = "DSA-1024"
ALG_SECP384R1_4 = "secp384r1"
ALG_RSAESPKCS1V15_5 = "RSAES-PKCS1-v1_5"
ALG_BENALOH_6 = "Benaloh"
ALG_SECP521R1_7 = "secp521r1"
ALG_ELGAMALSIGNATURE_8 = "ElGamal-signature"
ALG_DH_9 = "DH"
ALG_X25519_10 = "X25519"
ALG_DIFFIEHELLMANMERKLE_11 = "Diffie-Hellman-Merkle"
ALG_SECT283K1_12 = "sect283k1"
ALG_P384_13 = "P-384"
# TODO migrate prime256v1 to a post-quantum scheme (harvest-now-decrypt-later)
# TODO migrate ffdhe4096 to a post-quantum scheme (harvest-now-decrypt-later)
# TODO migrate X448 to a post-quantum scheme (harvest-now-decrypt-later)
# TODO migrate Goldwasser-Micali to a post-quantum scheme (harvest-now-decrypt-later)
# TODO migrate pairing-based to a post-quantum scheme (harvest-now-decrypt-later)
# TODO migrate RSA-512 to a post-quantum scheme (harvest-now-decrypt-later)
enabledAlgorithms = ["sect283k1", "Boneh-Lynn-Shacham", "MODP-6144", "BLS", "DSTU-4145", "Damgard-Jurik", "Rabin-Williams", "ElGamal"]
